ui/dashboards/src/components/PanelDrawer/PanelPreview.tsx

```diff
@@ -11,15 +11,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { useRef } from 'react';
 import { Box } from '@mui/material';
 import { QueryDefinition } from '@perses-dev/core';
 import { DataQueriesProvider } from '@perses-dev/plugin-system';
 import { PanelEditorValues } from '../../context';
 import { Panel } from '../Panel';
+import { useSuggestedStepMs } from '../../utils';
 
 const PANEL_PREVIEW_HEIGHT = 300;
+const PANEL_PREVIEW_DEFAULT_WIDTH = 800;
```

Comment:
setting the default width as 800 for now since our panel editor's width is fixed